app/frontend/src/App.vue

```diff
@@ -14,15 +14,21 @@
 </template>
 
 <script>
+// eslint-disable-next-line vue/no-reserved-component-names
```

Comment:
i dont think we need all these instructions telling eslint to ignore in this file.
I had them in the other repo because the components were named Header and Footer which are reserved names.
Where here they are BCGovHeader and BCGovFoter.
Could try removing.